### PR TITLE
test(chat-service): add test case for attachment file

### LINKED ISSUE #1395: @chat-service:add test case for has many relation of attachment File.
**Describe the bug**
add test case for has many relation of attachment File.

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
add test case of has many relation for attachment file

GH-1395

## Description

add test case of has many relation for attachment file
Fixes #1395 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/chat-service/src/__tests__/integration/message.repository.integration.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      .post('/messages')
+      .set('authorization', `Bearer ${token}`)
+      .send(messageObj);
+
+    const testFile = await client
+      .post('/attach-files')
+      .set('authorization', `Bearer ${token}`)
+      .send(dataMessageFile);
+    const filter = {
+      where: {id: testFile.body.id},
```
Comment (shubhamp-sf) [inline services/chat-service/src/__tests__/integration/message.repository.integration.ts:50]: Shouldn't this be message's id?

File: services/chat-service/src/__tests__/integration/message.repository.integration.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      include: [{relation: 'attachmentFiles'}],
+    };
+    const path = `/messages?filter=${encodeURIComponent(
+      JSON.stringify(filter),
+    )}`;
```
Comment (shubhamp-sf) [inline services/chat-service/src/__tests__/integration/message.repository.integration.ts:55]: Why are we using file's id in filter for `/messages` path?

Comment (Surbhi-sharma1) [reply services/chat-service/src/__tests__/integration/message.repository.integration.ts]: changes done

File: services/chat-service/src/__tests__/integration/message.repository.integration.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    const relationRes = await client
+      .get(path)
+      .set('Authorization', `Bearer ${token}`);
+    expect(relationRes.body).to.be.deepEqual([message.body]);
+  });
```
Comment (shubhamp-sf) [inline services/chat-service/src/__tests__/integration/message.repository.integration.ts:53]: The motive of this test case should be to check if setting `include: [{relation: 'attachmentFiles'}]` actually returns the related attachmentFiles. Which is currently not being checked.

**Review by shubhamp-sf — APPROVED** [review]

**Pull request merged**